src/v2/Components/ArtworkFilter/ArtworkFilterMobileActionSheet.tsx

```diff
@@ -108,7 +108,7 @@ export const ArtworkFilterMobileActionSheet: React.FC<{
       </Header>
 
       <Content ref={contentRef as any}>
-        <Box width="100%" p={2}>
+        <Box width="100%" p={2} pb={0}>
```

Comment:
I'm surprised that Box would have inherent padding without this explicit 0 value.

Reply:
It doesn't — but there's a p={2} here. It'd be better to just do `px={2} pt={2}`

Reply:
Ahh, that's right. Eyes skipped right over that first attribute.